demo/csi/aws/README.md

```diff
@@ -0,0 +1,11 @@
+# AWS CSI Plugins
+
+Configurations are provided here for [Elastic Block Store](https://aws.amazon.com/ebs/) and [Elastic Filesystem](https://aws.amazon.com/efs/) CSI drivers for use on [Amazon Web Services](https://aws.amazon.com).
+
+## Elastic Block Store
+
+EBS provides virtual block disks. Disks are tied to an availability zone, and aside from [io1/io2 multi-attach](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volumes-multi.html), which requires special filesystems to use in read-write mode, may only be used by a single node at a time.
```

Comment:
Last I checked the EBS plugin doesn't support the multi-writer mode. Might be worth omitting that reference or clarifying that the EBS plugin doesn't support it.